app/uk/gov/hmrc/accessibilitystatementfrontend/config/AppConfig.scala

```diff
@@ -36,10 +39,22 @@ case class AppConfig @Inject() (
   val reportAccessibilityProblemUrl          =
     s"$contactFrontendHostUrl/contact/accessibility-unauthenticated"
 
-  val showDraftStatementsEnabled: Boolean =
-    config
-      .getOptional[Boolean]("features.show-draft-statements")
-      .getOrElse(false)
+  val visibleStatuses: Set[Visibility] = {
+    config.getOptional[Seq[String]]("features.visibility") map (_.toSet) match {
+      case None               =>
+        logger.error("Config key not found: features.visibility, using default visibilities of: Public")
+        Set(Public)
+      case Some(visibilities) =>
+        visibilities.flatMap { visibility =>
+          visibilityParser.parse(visibility) match {
+            case Right(visibility) => Some(visibility)
+            case Left(error)       =>
+              logger.error(s"Invalid visibility status in config: $visibility, with error: $error")
```

Comment:
@JoPintoPaul I wonder if we should throw an exception here rather than logging an error?

Reply:
@oscarduignan As discussed, because injection 👍